.github/workflows/tests.yml

```diff
@@ -41,28 +41,40 @@ jobs:
       run: python scripts/lint.py rules/
 
   tests:
-    name: Tests in ${{ matrix.python }}
-    runs-on: ubuntu-20.04
+    name: Tests in ${{ matrix.python-version }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
     needs: [code_style, rule_linter]
     strategy:
       fail-fast: false
       matrix:
+        os: [ubuntu-20.4, windows-2019, macos-10.15]
```

Comment:
We pined the OS versions in https://github.com/fireeye/capa/commit/29d8f1fd27e72883ac32910731b958f0e40be7ae, so I have pined them here while rebasing